automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/cluster/ClusterExceptionSpec.scala

```diff
@@ -65,19 +65,6 @@ class ClusterExceptionSpec extends FreeSpec with LeonardoTestUtils with Parallel
       }
     }
 
-    "should throw 401 for invalid token" in {
```

Comment:
This looks like the right place for this tests, why are we moving it?

Reply:
All the rest of the tests use only Cluster logic, but this one also uses Notebooks logic, so I moved it to one of the notebook specs